src/home_robot_hw/home_robot_hw/env/simple_navigation_env.py

```diff
@@ -0,0 +1,55 @@
+from typing import Any, Dict, Optional
+
+import numpy as np
+import rospy
+
+import home_robot
+from home_robot_hw.env.stretch_abstract_env import StretchEnv
+
+class StretchSimpleNavEnv(StretchEnv):
+    """Create a detic-based object nav environment"""
+
+    def __init__(
+        self, config=None, forward_step=0.25, *args, **kwargs
+    ):
+        super().__init__(*args, **kwargs)
+
+        self.forward_step = forward_step  # in meters
+        self.reset()
+
+    def reset(self):
+        self._current_pose = self.get_base_pose()
+
+    def apply_action(self):
+        continuous_action = np.zeros(3)
+        continuous_action[0] = 0.25
+        if continuous_action is not None:
```

Comment:
Why are you checking if it's None if you just assigned values to it?

Reply:
Leftover from old code which was receiving an input. Will remove this.